From 12b69163841eae8c3d1f86039520fde8c0c43f25 Mon Sep 17 00:00:00 2001
From: Rick Farrington <rfarrington@marvell.com>
Date: Fri, 26 Jun 2020 15:32:04 -0400
Subject: [PATCH 1/5] drivers: marvell: otx2: enable prerequisite items for RAS
 GHES

commit ab7ea489e2d2b3de94db499264ccdd47ac0fc948 from
git@git.assembla.com:cavium/WindRiver.linux.git

Change-Id: Iaa9746bddc6d5013f398d833d2997780ae43b282
Signed-off-by: Rick Farrington <rfarrington@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/c/IP/SW/kernel/linux/+/30960
Tested-by: sa_ip-sw-jenkins <sa_ip-sw-jenkins@marvell.com>
Reviewed-by: Chandrakala Chavva <cchavva@marvell.com>
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/soc/marvell/Kconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/drivers/soc/marvell/Kconfig b/drivers/soc/marvell/Kconfig
index 3c81c2af9a93..0c666c60a452 100644
--- a/drivers/soc/marvell/Kconfig
+++ b/drivers/soc/marvell/Kconfig
@@ -85,7 +85,9 @@ config MDIO_DEBUGFS
 
 config OCTEONTX2_SDEI_GHES
 	bool "OcteonTX2 Generic Hardware Error Source (GHES) support"
-	depends on ARM_SDE_INTERFACE
+	select ARM_SDE_INTERFACE
+	select ACPI_APEI
+	select ACPI_APEI_GHES
 	help
 	  Select this option to enable support for RAS Generic Hardware Error
 	  Source (GHES) reporting.
-- 
2.17.1

